src/test/regress/expected/intermediate_result_pruning.out

```diff
@@ -4,6 +4,8 @@ SET citus.log_intermediate_results TO TRUE;
 SET citus.shard_count TO 4;
 SET citus.next_shard_id TO 1480000;
 SET citus.shard_replication_factor = 1;
+SET citus.enable_local_execution_planning TO 1;
+SET citus.enable_local_execution TO 1;
```

Comment:
Can you add some tests to confirm using citus.log_remote_commands to confirm that local execution is being used?

Reply:
makes sense, we've another GUC, which logs local execution even on `DEBUG1`, `SET citus.log_local_commands TO TRUE;`, adding that now

Reply:
btw, In general, I found the number of tests that explicitly tests the edge cases of the PR is not here, so adding another test file with such queries